src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/CatchParameterNameCheck.java

```diff
@@ -35,16 +35,16 @@
  * <li>allows {@code e} abbreviation (suitable for exceptions end errors)</li>
  * <li>allows {@code ex} abbreviation (suitable for exceptions)</li>
  * <li>allows {@code t} abbreviation (suitable for throwables)</li>
+ * <li>allows {@code _} for unnamed catch parameters</li>
  * <li>prohibits numbered abbreviations like {@code e1} or {@code t2}</li>
  * <li>prohibits one letter prefixes like {@code pException}</li>
  * <li>prohibits two letter abbreviations like {@code ie} or {@code ee}</li>
- * <li>prohibits any other characters than letters</li>
```

Comment:
If we are going to remove this then there needs to be something equivalent that mentions the rest of the prohibitions not mentioned above. This statement covered those before.

For example, we don't say `$` is prohibited, nor underscores (excluding `_` alone) nor numbers. We say numbered abbreviations above, but its not just abbreviations, it is all numbers.

Reply:
prohibits any other characters than letters and underscore?

Reply:
> prohibits any other characters than letters and underscore?

The concern with just saying "underscore" is you have to make sure they understand underscore with anything else is disallowed. It is just a single `_` which is allowed.

Truthfully, I think original wording is fine. We say first we allow `_`. We can say we don't allow anything other than letters. It should be understood it is top to bottom order. Allows overrides the prohibits.

Reply:
> Truthfully, I think original wording is fine. We say first we allow `_`. We can say we don't allow anything other than letters. It should be understood it is top to bottom order. Allows overrides the prohibits.

make sense. Original wording is restored